modules/nw-sriov-network-attachment.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="configuring-sr-iov-networks_{context}"]
+= Configuring SR-IOV additional network
+
+You can configure an additional network that uses SR-IOV hardware by creating a SriovNetwork Custom Resource (CR).
+When you create a SriovNetwork CR, the SR-IOV Operator automatically creates a NetworkAttachmentDefinition CR.
+
+[NOTE]
+=====
+Do not modify or delete a SriovNetwork Custom Resource (CR) if it is attached to any Pods in the `running` state.
+=====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following SriovNetwork CR, and then save the YAML in the `<name>-sriov-network.yaml` file. Replace `<name>` with a name for this additional network.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: <name> <1>
+  namespace: sriov-network-operator <2>
+spec:
+  networkNamespace: <target_namespace> <3>
+  ipam: |- <4>
+    ...
+  vlan: <vlan> <5>
+  resourceName: <sriov_resource_name> <6>
+----
+<1> Replace `<name>` with a name for the CR. The Operator will create a NetworkAttachmentDefinition CR with same name.
+<2> Specify the namespace where the SR-IOV Operator is installed.
+<3> Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
+<4> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<5> Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
+<6> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
+
+. Create the CR by running the following command:
++
+----
+$ oc create -f <filename> <1>
+----
+<1>  Replace `<filename>` with the name of the file you created in the previous step.
```

Comment:
better add one step to check the NetworkAttachmentDefinition is created in target namespace
oc get net-attach-def -n $target_namespace

Reply:
Good idea!